SingularityService/src/main/docker/start.sh

```diff
@@ -15,7 +15,12 @@ DEFAULT_URI_BASE="http://${SINGULARITY_HOSTNAME:=localhost}:${SINGULARITY_PORT:=
 
 args+=( -Xmx${SINGULARITY_MAX_HEAP:-512m} )
 args+=( -Djava.net.preferIPv4Stack=true )
+args+=( -Ddw.server.applicationContextPath="${SINGULARITY_APP_CTX:=/singularity}" )
```

Comment:
So, we need two different ones. There is the app context path, and the uri base (host + path that the ui uses for config). We can reuse the `SINGULARITY_UI_BASE` var that we use to calculate the default uri base (or set them both to `SINGULARITY_APP_CTX` which might even make more sense)